lib/core/router/handle_redirect.dart

```diff
@@ -68,7 +71,11 @@ String? handleRedirect(Ref ref, GoRouterState state) {
     final mappedLocation = _mapTwitterPath(state);
     if (mappedLocation != null) return mappedLocation;
 
-    // if the location doesn't exist navigate to home instead
+    // if the location doesn't exist, launch it and navigate to home instead
+    final launcher = ref.watch(launcherProvider);
+    launcher(state.location).onError((error, stackTrace) {
+      log('failed to launch url: $error');
+    });
```

Comment:
- `state.location` only includes the path, we need to prefix the twitter url
- the error is handled by the launcherProvider, so no need for the onError handler

Reply:
Updated. Please check and advice.